dalme_app/static/css/dalme_public/dalme_public.css

```diff
@@ -1267,8 +1266,8 @@ i.inactive {
   color: #4e0f0f;
   background: #f1b5b5;
 }
-#diva_viewer {
-    flex-grow: 1;
+.diva-wrapper {
```

Comment:
There is another rule for this class (`div.diva-wrapper`) around line 1964. Adding `min-height: calc(100vh - (var(--header-height) + 96px));` (or 88px, see my other comment on this file), to solve issue A in my PR comment.